src/logger.js

```diff
@@ -0,0 +1,21 @@
+"use strict";
```

Comment:
Need a copyright notice for a new file (© you since you wrote it).

Also we don't need `use strict` anymore.